docs/en/observability/apm/advanced-setup/index.asciidoc

```diff
@@ -0,0 +1,30 @@
+
+[[apm-setting-up-and-running]]
+= APM Server advanced setup
+
+++++
+<titleabbrev>Advanced setup</titleabbrev>
+++++
```

Comment:
I almost wonder if this should be a subsection of "Configure APM Server" instead

Reply:
I moved the [Advanced setup section](https://observability-docs_bk_4230.docs-preview.app.elstc.co/guide/en/observability/master/apm-setting-up-and-running.html) into "Configure APM Server". Take a look and let me know what you think -- it may get a bit lost, but maybe that's ok since it's intended to be used by only some advanced users.

Reply:
> it may get a bit lost, but maybe that's ok since it's intended to be used by only some advanced users.

Agreed. I like it.